cmd/frostfs-cli/modules/container/create.go

```diff
@@ -209,8 +212,11 @@ func parseAttributes(dst *container.Container, attributes []string) error {
 		container.SetCreationTime(dst, time.Now())
 	}
 
-	if containerName != "" {
-		container.SetName(dst, containerName)
+	if containerNnsName != "" {
+		dst.SetAttribute(containerApi.SysAttributeName, containerNnsName)
```

Comment:
Can we use `containerSDK.WriteDomain` here?

Reply:
Thanks, replaced.